released/packages/coq-simple-io/coq-simple-io.1.1.0/opam

```diff
@@ -0,0 +1,38 @@
+opam-version: "2.0"
+maintainer: "Li-yao Xia <lysxia@gmail.com>"
+authors: "Li-yao Xia"
+homepage: "https://github.com/Lysxia/coq-simple-io"
+bug-reports: "https://github.com/Lysxia/coq-simple-io/issues"
+license: "MIT"
+dev-repo: "git+https://github.com/Lysxia/coq-simple-io.git"
+build: [
+  [make "build"]
+  [make "test"] {with-test}
+]
+install: [make "install"]
+remove: [make "uninstall"]
+depends: [
+  "ocaml"
+  "coq"
+  "coq-ext-lib" {>= "0.10.0"}
+  "ocamlbuild" {with-test & >= "0.9.0"}
+]
+tags: [
+  "date:2019-05-05"
+  "logpath:SimpleIO"
+  "keyword:extraction"
+  "keyword:effects"
+]
+synopsis: "IO monad for Coq"
+description: """
+This library provides tools to implement IO programs directly in Coq, in a
+similar style to Haskell. Facilities for formal verification are not included.
+
+IO is defined as a parameter with a purely functional interface in Coq,
+to be extracted to OCaml. Some wrappers for the basic types and functions in
+the OCaml Pervasives module are provided. Users are free to define their own
+APIs on top of this IO type."""
+url {
+  src: "https://github.com/Lysxia/coq-simple-io/archive/1.1.0.tar.gz"
+  checksum: "sha256=888749c06e58b65022902b051993a1f49d38c113480586039f20d159ba7a41b8"
```

Comment:
I get a different SHA256 checksum when downloading this archive (`6e25d225e12f1b85346dff7053e4c93f523d573255827bfba6aa0a4a4b85b862`). Please adjust the package to the correct checksum (this is what kills the CI builds).